src/docs/markdown/caddyfile/directives/reverse_proxy.md

```diff
@@ -557,7 +557,7 @@ transport http {
 
 - **tls_timeout** <span id="tls_timeout"/> is the maximum [duration](/docs/conventions#durations) to wait for the TLS handshake to complete. Default: No timeout.
 
-- **tls_trusted_ca_certs** <span id="tls_trusted_ca_certs"/> is a list of PEM files that specify CA public keys to trust when connecting to the backend.
+- **tls_trust_pool** <span id="tls_trust_pool"/> configures the source of trusted certificate authorities similar to the [`trust_pool` sub-directive](docs/caddyfile/directives/tls#trust_pool) described on the `tls` directive documentation. The list of trust pool sources available in standard Caddy installation is available [here](docs/caddyfile/directives/tls#trust-pool-providers).
```

Comment:
Links should either have a leading `/`, or just `tls#anchor` since that's properly relative to this file. A relative path like this won't work from this file.

Reply:
I missed the leading slash as I was cleaning the path 🤦🏼‍♂️ I'll fix it. The link is to a different file, i.e. the TLS directive page.

Reply:
Yeah - `tls#anchor` would work too because both the current and target are in the same directory, but absolute path is fine